File: C++/k번째수.cpp
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer, tmp;
    int comLen = commands.size();
    
    for(int i = 0 ; i < comLen ; i++){
        tmp = array;
        sort(tmp.begin() + commands[i][0] - 1, tmp.begin() + commands[i][1]);
        answer.push_back(tmp[commands[i][0] + commands[i][2] - 2]);
    }
    
    return answer;
}

File: python/k번째수.py
def solution(array, commands):
    answer = []
    
    for i in range(0,len(commands)):
        a = array[commands[i][0] - 1 : commands[i][1]]
        a.sort()
        answer.append(a[commands[i][2] - 1])
        
    return answer